libkbfs/key_manager.go

```diff
@@ -450,21 +450,41 @@ func (km *KeyManagerStandard) Rekey(ctx context.Context, md *RootMetadata, promp
 			return false, nil, err
 		}
 		resolvedInfo := resolvedHandle.ConflictInfo()
-		if latestHandle.ConflictInfo != resolvedInfo {
-			km.log.CDebugf(ctx, "handle for %s is conflicted",
-				handle.GetCanonicalPath())
-			if resolvedInfo != nil {
-				// ConflictInfo is inconsistent.
+		if resolvedInfo == nil {
```

Comment:
Moved it into `TlfHandle.UpdateConflictInfo()`, replacing `TlfHandle.SetConflictInfo()`. As a bonus, this gets some test coverage for this.

@aalness should the same treatment be done for `SetFinalizedInfo`? If so, I can do that in another PR.